edit-post/README.md

```diff
@@ -55,48 +53,31 @@ Title displayed at the top of the sidebar.
 - Required: Yes
 
 
-### `PluginMoreMenuItem`
-**Experimental**
+### `PluginSidebarMoreMenuItem`
 
-Renders a menu item in the more menu drop down, and can be used to activate other plugin UI components.
+Renders a menu item in `Plugins` group in `More Menu` drop down, and can be used to activate the corresponding `PluginSidebar` component.
 The text within the component appears as the menu item label.
 
 _Example:_
 
 ```jsx
-const { PluginMoreMenuItem } = wp.editPost;
+const { PluginSidebarMoreMenuItem } = wp.editPost;
 
-const MyPluginMenuItem = () => (
-	<PluginMoreMenuItem
-		name="my-plugin"
-		icon="yes"
-		type="sidebar"
+const MySidebarMoreMenuItem = () => (
+	<PluginSidebarMoreMenuItem
 		target="my-sidebar"
+		icon="yes"
 	>
-		My Sidebar
-	</PluginMoreMenuItem>
+		My sidebar title
+	</PluginSidebarMoreMenuItem>
 );
 ```
 
 #### Props
 
-##### name
-
-A string identifying the menu item. Must be unique for every menu item registered within the scope of your plugin.
```

Comment:
I didn't find any reason to keep it for the time being. We can bring it back as soon there is use case for it.